### PR TITLE
Creating new conflict resolution document

### PR DESCRIPTION
**Pull request by wareid**
Taking the content from dispute resolution and structurin it into an HTML document.

### CODE REVIEW

File: conflict_resolution.html [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+                <li>
+                    ...
+                </li>
+            </ol>
+        </section>
+
+        <section id="ombudspeople">
+            <h2>The Role of the Ombudsperson</h2>
+
+            <p>An Ombudsperson can advise or assist you with:</p>
```
Comment (rachelyager-nyc) [inline conflict_resolution.html:105]: This section clarifies the role of the Ombudsperson as a designated role to "advise or assist" in the conflict resolution process. It may be helpful to explain further that the Ombudsperson cannot take sides or provide an opinion about the conflict at this beginning stage of conflict resolution. The Ombudsperson must maintain neutrality. 
My suggestion is to include a paragraph outlining what the Ombudsperson's role does not entail, in order to avoid any potential confusion or misunderstandings.

**Comment by rachelyager-nyc** [comment]
@wareid  I added a comment on the document regarding the role of the Ombudsperson. Sharing it here for discussion:

This section clarifies the role of the Ombudsperson as a designated role to "advise or assist" in the conflict resolution process. It may be helpful to explain further that the Ombudsperson cannot take sides or provide an opinion about the conflict at this beginning stage of conflict resolution. The Ombudsperson must maintain neutrality.

**I suggest including a paragraph outlining what the Ombudsperson's role does not entail** in order to avoid any potential confusion or misunderstandings.

File: conflict_resolution.html [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+
+            <p>The procedure for addressing a conflict will include some or all of the following steps:</p>
+
+            <ol>
+                <li>Incident is reported.
+                    <ul>
+                        <li>Reporting can take on many forms, including reaching out to the person directly, reporting the issue to a chair or ombudsperson, or to a member of the team.</li>
+                    </ul>
+                </li>
+                <li>Incident is discussed, impact is shared, and first resolution is proposed.
```
Comment (rachelyager-nyc) [inline conflict_resolution.html:91]: Should we provide a mechanism by which we can **store a record** of this reported incident and the outcome?

**Comment by rachelyager-nyc** [comment]
Is there a way to enable **anonymous reporting of incidents or unacceptable behavior**? 
 
 This anonymity can be provided by an individual or a group of people who have witnessed the conflict but were not directly involved.

File: conflict_resolution.html (hunk 2 above)
Comment (jenstrickland) [reply conflict_resolution.html:91]: That makes me think… I wonder if there could be an online form to report, with an option of an anonymous report, as well. If it was an anonymous report, then the process would be different, but I can see some might feel more comfortable to report anonymously.

**Comment by jenstrickland** [comment]
Consider adding a 'Table of Contents' to the top of the page that will link to the individual sections. At this time, this is the outline:

```
<h2>Table of Contents</h2>
<ul>
  <li>Introduction</li>
  <li>Get advice from an Ombudsperson</li>
  <li>Mediation - when it's effective
    <ul>
      <li>Individual or facilitated mediation</li>
       <li>Professional mediation</li>
    </ul>
  </li>
  <li>Investigations
    <ul>
       <li>Timelines</li>
    </ul>
  </li>
  <li>Disciplinary actions</li>
</ul>
```

File: conflict_resolution.html [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+            <p>When a conflict arises it is important for everyone involved to know what will happen and what resources are available to them. Any time a Code of Conduct violation or complaint occurs it is important that those involved, especially the party/parties affected by the violation, feel supported and respected.</p>
+
+            <p>How a conflict is handled will depend on a number of factors including the type of complaint, severity, frequency, and contributing factors (for example, has anyone been involved in conflicts before, is there a pattern?).</p>
+
+            <p>The procedure for addressing a conflict will include some or all of the following steps:</p>
+
+            <ol>
+                <li>Incident is reported.
+                    <ul>
+                        <li>Reporting can take on many forms, including reaching out to the person directly, reporting the issue to a chair or ombudsperson, or to a member of the team.</li>
```
Comment (dbooth-boston) [inline conflict_resolution.html:88]: Reaching out to the person directly is a step that can be taken, but it is not a form of reporting.   It doesn't belong in the list of ways to report the incident.

**Review by dbooth-boston — COMMENTED** [review]
This document should be about how to report and resolve violations of the CoC, and needs to be named, titled and framed that way.  Framing it as "conflict resolution" is misleading, because that isn't what it is about.   People can have conflicts or disagreements without any kind of CoC violation, and CoC violations do not necessarily involve conflicts.  Indeed, many CoC violations are unintentional and do not involve conflict at all.  They are frequently caused by a lack of understanding (on the part of the violator) with no malicious intent whatsoever.  It is misleading to frame them as conflicts.

File: conflict_resolution.html (hunk 2 above)
Comment (wareid) [reply conflict_resolution.html:91]: This is something we're looking at, but won't be covered extensively in this document, but it is part of it!

**Comment by wareid** [comment]
@rachelyager-nyc @dbooth-boston I have made some changes in accordance with your feedback. I believe more work is needed, but if you're happy with the current state, I'd like to merge this so we can review the much more readable version and solicit more feedback.

File: conflict_resolution.html [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+        <section id="abstract">
+            <p>This document is a companion to the W3C Code of Conduct which clarifies the procedures for handling incidents relating to Code of Conduct violations or complaints.</p>
+        </section>
+
+        <section id="sotd"></section>
+
+        <section id="introduction">
+            <h2>Introduction</h2>
+
+            <p>As we engage with one another, incidents can happen. Incidents can take many forms, but participants have the right to a safe and positive work environment. The goal of conflict resolution is that participants in W3C can:</p>
```
Comment (dbooth-boston) [inline conflict_resolution.html:44]: +1 to the word "incident".  Excellent word choice.

Suggest: s/conflict resolution/this incident resolution procedure/

File: conflict_resolution.html [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+
+            <ul>
+                <li>Get help and support to safely resolve issues that negatively affect them</li>
+                <li>Correct their behaviours that negatively affect other participants in W3C</li>
```
Comment (dbooth-boston) [inline conflict_resolution.html:48]: s/their//

File: conflict_resolution.html [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+                <li>Continue to safely work and contribute to the W3C after an issue has been raised</li>
+            </ul>
+                
+            <p>The Code of Conduct is the standard that all W3C participants are required to follow and is used as the basis for any resolution and disciplinary actions. In exceptional circumstances, individuals may have their W3C participation suspended or withdrawn as a result of a complaint or investigation.</p>
+                
+            <p>All situations are different and W3C participants are encouraged to use whatever method of conflict resolution they feel comfortable with.</p>
```
Comment (dbooth-boston) [inline conflict_resolution.html:54]: s/whatever/whichever/, since we don't want participants taking matters into their own hands inappropriately.

s/conflict resolution/incident resolution/

File: conflict_resolution.html [hunk 7]
```diff
@@ -0,0 +1,200 @@
...
+            <p>Participants are encouraged to try and resolve issues themselves in the first instance. However, if participants do not feel comfortable or safe doing so, they can get help through an ombudsperson, facilitators and/or mediation.</p>
+                
+            <p>Depending on the severity of the situation, it may not be appropriate to attempt to resolve the issue informally or in the context of the group. Participants should immediately report an incident whenever they feel unsafe or threatened by the behaviour or actions of other W3C participants.</p>
+                
+            <p>Everyone involved in the conflict resolution process is expected to:</p>
+            
+            <ul>
+                <li>Show respect for others</li>
+                <li>Work together to resolve the complaint</li>
+                <li>Maintain confidentiality</li>
```
Comment (dbooth-boston) [inline conflict_resolution.html:65]: I think we need to be more specific here.  I don't think we want to require everyone to maintain confidentiality all the time.  I think the confidentiality requirement mostly applies to the ombuds and others who are in a position of W3C responsibility for responding to complaints of CoC violations.   Presumably a person who experienced a CoC violation should have the option of reporting it publicly, and anyone who is accused of a CoC violation should have the option of reporting that accusation publicly.

File: conflict_resolution.html [hunk 8]
```diff
@@ -0,0 +1,200 @@
...
+            <p>A disciplinary investigation can be started where one of the following apply:</p>
+                
+            <ul>
+                <li>It’s not appropriate to begin with mediation</li> 
+                <li>All other mediation or resolution attempts have not worked</li>
+            </ul>
+        </section>
+
+        <section id="procedures">
+            <h2>Conflict Resolution Procedures</h2>
```
Comment (dbooth-boston) [inline conflict_resolution.html:77]: s/Conflict Resolution/Incident Resolution/

File: conflict_resolution.html [hunk 9]
```diff
@@ -0,0 +1,200 @@
...
+
+            <p>When an incident arises it is important for everyone involved to know what will happen and what resources are available to them. Any time a Code of Conduct violation or complaint occurs it is important that those involved, especially the party/parties affected by the incident, feel supported and respected.</p>
+
+            <p>How a conflict is handled will depend on a number of factors including the type of complaint, severity, frequency, and contributing factors (for example, has anyone been involved in incidents before, is there a pattern?).</p>
```
Comment (dbooth-boston) [inline conflict_resolution.html:81]: s/a conflict/an incident/

s/type of complaint/type of incident/

File: conflict_resolution.html [hunk 10]
```diff
@@ -0,0 +1,200 @@
...
+
+            <p>The procedure for addressing a conflict will include some or all of the following steps:</p>
```
Comment (dbooth-boston) [inline conflict_resolution.html:83]: s/a conflict/an incident/

File: conflict_resolution.html [hunk 11]
```diff
@@ -0,0 +1,200 @@
...
+
+            <ol>
+                <li>Incident is reported.
+                    <ul>
+                        <li>Reporting can take on many forms including reporting the issue to a chair, ombudsperson, through an anonymous report, or a member of the team.</li>
+                    </ul>
+                </li>
+                <li>Incident is discussed, impact is shared, and first resolution is proposed.
+                    <ul>
+                        <li>For conflicts that have a clear path to resolution and low severity (for example, a harsh criticism in a meeting or inappropriate language), the focus should be on coming to a shared understanding of what happened and agreeing to a path to resolution (for example, an apology or editing a comment or minutes).</li>
```
Comment (dbooth-boston) [inline conflict_resolution.html:93]: s/conflicts/incidents/

File: conflict_resolution.html [hunk 12]
```diff
@@ -0,0 +1,200 @@
...
+                <li>
+                    ...
+                </li>
+            </ol>
+        </section>
+
+        <section id="ombudspeople">
+            <h2>The Role of the Ombudsperson</h2>
+
+            <p>Ombudspeople are advisors to participants who can provide guidance, advice, and support during the conflict resolution process. Ombudspeople are meant to be neutral third parties to any incident, and as such are required not to take sides or provide opinions on the indicidents they are involved in.</p>
```
Comment (dbooth-boston) [inline conflict_resolution.html:105]: I certainly agree with the need for Ombuds to be neutral.  But do we really want Ombuds to not provide opinions on the incidents that are reported to them?  I think this last sentence needs more consideration.  It seems to me that:

- It will be nearly impossible to prevent an Ombud from expressing an opinion -- especially since it is very important for an Ombud to be empathetic; and
- An Ombud's professional opinion could be enormously helpful to those who are involved in an incident.

Also, spelling error: s/indicidents/incidents/

File: conflict_resolution.html [hunk 13]
```diff
@@ -0,0 +1,200 @@
...
+                <li>How to raise a formal complaint</li>
+            </ul>
+
+            <p>TODO: More about the role of the ombudsperson and what they can and cannot do.</p>
+        </section>
+
+        <section id="mediation">
+            <h2>Mediation</h2>
+
+            <p>Mediation is most effective when used earlier in a dispute, before people become fixed in their position, as it provides a positive opportunity to resolve practical problems.</p>
```
Comment (dbooth-boston) [inline conflict_resolution.html:121]: s/earlier/early/

File: conflict_resolution.html [hunk 14]
```diff
@@ -0,0 +1,200 @@
...
+
+            <p>It empowers people to work together to resolve issues, and can therefore be less stressful and have a less negative impact on individuals. Mediation can also be successful in later stages after an investigation has concluded to help people rebuild relationships.</p>
```
Comment (dbooth-boston) [inline conflict_resolution.html:123]: Less stressful than what?   I suggest shortening that first sentence: 
s/, and can therefore be less stressful and have a less negative impact on individuals//

Also add commas: "Mediation can also be successful in later stages, after an investigation has concluded, to help people rebuild relationships."

File: conflict_resolution.html [hunk 15]
```diff
@@ -0,0 +1,200 @@
...
+                <li>Formal Mediation: This approach involves the parties involved, as well as a professional mediator. It may also include parties like ombudspeople or chairs to help facilitate or provide necessary context or support. A professional mediator will be an independent third-party who is brought in to find a resolution to the conflict.</li>
+            </ul>
+
+            <section id="professional-mediation">
+                <h3>Professional Mediation</h3>
+
+                <p>In cases where it's deemed necessary to bring in a professional mediator, all parties involved:</p>
+
+                <ul>
+                    <li>will agree to attend mediation as an option for resolving the issue</li>
```
Comment (dbooth-boston) [inline conflict_resolution.html:140]: This says "will agree", but we should clarify: Is participation in mediation optional or required?

File: conflict_resolution.html [hunk 16]
```diff
@@ -0,0 +1,200 @@
...
+                    <li>will individually attend a meeting with independent trained mediators</li>
+                    <li>will collectively attend a meeting to state how each party sees the problem and how it might be resolved</li>
```
Comment (dbooth-boston) [inline conflict_resolution.html:142]: I suggest deleting the above two bullets (about individually and collectively attending meetings).  I don't think we should dictate how the professional mediation should be conducted.  Professional mediators should know better than us about how to make the mediation process effective.

File: conflict_resolution.html [hunk 17]
```diff
@@ -0,0 +1,200 @@
...
+            <h2>Conflict Resolution Procedures</h2>
+
+            <p>When an incident arises it is important for everyone involved to know what will happen and what resources are available to them. Any time a Code of Conduct violation or complaint occurs it is important that those involved, especially the party/parties affected by the incident, feel supported and respected.</p>
+
+            <p>How a conflict is handled will depend on a number of factors including the type of complaint, severity, frequency, and contributing factors (for example, has anyone been involved in incidents before, is there a pattern?).</p>
+
+            <p>The procedure for addressing a conflict will include some or all of the following steps:</p>
+
+            <ol>
+                <li>Incident is reported.
```
Comment (dbooth-boston) [inline conflict_resolution.html:86]: Before the "Incident is reported" item, I think it would be good to insert a step for "Incident is discussed directly with the parties involved".  I think it would also be helpful to provide a little guidance about how to do this most effectively.  For example, in the case of a person who inadvertently make someone else feel bad or slighted, I think it is helpful to both: 1. explain how it made the recipient feel and why; and 2. request a specific different approach in the future.   I think[ the example given in CoC section 5](https://w3c.github.io/PWETF/#mistake) is pretty good, though it leaves out (as implied) the component of explaining how the incident made the recipient feel and why:

> Alice: “Yeah I used X and it was really crazy!” Eve: “Hey, could you not use that word? What about ‘ridiculous’ instead?” Alice: “oh sorry, sure.” -> edits old message to say “Yeah I used X and it was really confusing!”

File: conflict_resolution.html [hunk 18]
```diff
@@ -0,0 +1,200 @@
...
+                    <ul>
+                        <li>Reporting can take on many forms including reporting the issue to a chair, ombudsperson, through an anonymous report, or a member of the team.</li>
```
Comment (dbooth-boston) [inline conflict_resolution.html:88]: Somewhere I think it would be good to provide guidance about when to report an incident, and encourage reporting.  Not every incident needs to be reported, but it is helpful if incidents are reported even when no further action is expected, in order to help detect patterns and repeat offenders.

File: conflict_resolution.html [hunk 19]
```diff
@@ -0,0 +1,200 @@
...
+        <section id="ombudspeople">
+            <h2>The Role of the Ombudsperson</h2>
+
+            <p>Ombudspeople are advisors to participants who can provide guidance, advice, and support during the conflict resolution process. Ombudspeople are meant to be neutral third parties to any incident, and as such are required not to take sides or provide opinions on the indicidents they are involved in.</p>
+
+            <p>An Ombudsperson can advise or assist you with:</p>
+
+            <ul>
+                <li>How the Code of Conduct may apply in your situation</li>
+                <li>How to start the mediation process</li>
```
Comment (dbooth-boston) [inline conflict_resolution.html:111]: s/the mediation/a mediation/

File: conflict_resolution.html [hunk 20]
```diff
@@ -0,0 +1,200 @@
...
+                
+            <p>It is usually between two parties, but mediation can also work for groups. Mediation can be used to resolve a range of issues including a breakdown in relationships or ongoing conflicts between group members.</p>
+
+            <p>Where it's appropriate to attempt to resolve an issue with mediation, there are two approaches that can be taken:</p>
+
+            <ul>
+                <li>Informal Mediation: This approach may include solely the parties involved in the incident, or extend to trusted individuals to help facilitate the conversation (for example, chairs or ombudspeople). The process may be more informal in that it takes place over a call or email correspondance, but the goal remains: resolution of conflict.</li>
+                <li>Formal Mediation: This approach involves the parties involved, as well as a professional mediator. It may also include parties like ombudspeople or chairs to help facilitate or provide necessary context or support. A professional mediator will be an independent third-party who is brought in to find a resolution to the conflict.</li>
```
Comment (dbooth-boston) [inline conflict_resolution.html:131]: Suggest re-ordering the wording of the first sentence to mention the professional mediator first (since that's the key point): "This approach involves a professional mediator, as well as the parties involved in the incident."

File: conflict_resolution.html [hunk 21]
```diff
@@ -0,0 +1,200 @@
...
+
+            <p>It empowers people to work together to resolve issues, and can therefore be less stressful and have a less negative impact on individuals. Mediation can also be successful in later stages after an investigation has concluded to help people rebuild relationships.</p>
+                
+            <p>It is usually between two parties, but mediation can also work for groups. Mediation can be used to resolve a range of issues including a breakdown in relationships or ongoing conflicts between group members.</p>
+
+            <p>Where it's appropriate to attempt to resolve an issue with mediation, there are two approaches that can be taken:</p>
+
+            <ul>
+                <li>Informal Mediation: This approach may include solely the parties involved in the incident, or extend to trusted individuals to help facilitate the conversation (for example, chairs or ombudspeople). The process may be more informal in that it takes place over a call or email correspondance, but the goal remains: resolution of conflict.</li>
```
Comment (dbooth-boston) [inline conflict_resolution.html:130]: If it only involves the parties involved it's direct discussion between the parties, not mediation, so I suggest rewording the above item to:

"Informal Mediation: Any mutually trusted individual may act as an informal mediator, such as a chair or Ombud or even another W3C participant.  The process may take place in person, over a call, through email, or through some other medium, but the goal remains: to reach resolution.

File: conflict_resolution.html [hunk 22]
```diff
@@ -0,0 +1,200 @@
...
+                </ul>
+
+                <p>TODO: Details of how to access mediation</p>
+            </section>
+        </section>
+
+        <section id="investigations">
+            <h2>Investigations</h2>
+
+            <p>If resolution attempts have not worked or the issue is not appropriate for mediation, a participant can raise a formal complaint against another participant.</p>
```
Comment (dbooth-boston) [inline conflict_resolution.html:153]: s/mediation/direct resolution or mediation/

s/can raise/may raise/

File: conflict_resolution.html [hunk 23]
```diff
@@ -0,0 +1,200 @@
...
+            <p>An investigator will be appointed and will talk to both parties and anyone else relevant. The investigator's role is to:</p>
+
+            <ul>
+                <li>establish what happened, in a fair and objective manner</li>
+                <li>ensure that the investigation is proportionate to the seriousness and complexity of the complaint</li>
+                <li>ensure that the investigation is concluded without delay</li>
+                <li>keep notes and submitted evidence in confidence</li>
+            </ul>
+
+            <p>The investigator will write a summary of the result of the investigation and recommend what actions should be taken. Outcomes and actions will be agreed by W3M and communicated to both parties.</p>
```
Comment (dbooth-boston) [inline conflict_resolution.html:175]: The first sentence in the line above is also about what the investigator is supposed to do, so it should be included in the above bullets instead of being a separate sentence:

```
<li>write a summary of the result of the investigation</li>
<li>recommend what actions should be taken</li>
```

File: conflict_resolution.html [hunk 24]
```diff
@@ -0,0 +1,200 @@
...
+            <p>There are various actions that the investigator can recommend, depending on the frequency and severity of the behaviours in conflict with the CoC.</p>
+
+            <p>The intent of the conflict resolution procedures is to ensure that individuals can continue to participate in W3C activities.</p>
+
+            <p>Where it is beneficial to the continued functioning of W3C, a high level summary of results and actions from an investigation may be communicated to the membership, for example if an individual is suspended from a working group or from being a chair in any working group. Otherwise, results and actions should be kept confidential.</p>
+
+            <p>Chairs, ombudspeople, mediators, and investigators can recommend the following actions be taken if resolution is not possible:</p>
+
+            <ol>
+                <li>Corrective action. The individual who had the complaint raised against them will be advised to correct their behaviour going forward, for example not engaging in the same behaviours that resulted in the complaint.</li>
```
Comment (dbooth-boston) [inline conflict_resolution.html:190]: s/had the complaint raised against them/violated the CoC/

File: conflict_resolution.html [hunk 25]
```diff
@@ -0,0 +1,200 @@
...
+                <li>the grounds for the complaint (for example, what parts of the CoC you think are relevant)</li>
+                <li>if appropriate, what you have done so far to try to resolve the complaint (for example, trying to solve the issue yourself or through mediation)</li>
+                <li>any evidence or applicable materials relating to the conflict (e.g., emails, minutes, screenshots, or other materials)</li>
+                <li>what resolution or action you wish to be taken to address the conflict</li>
+            </ul>
+
+            <p>An investigator will be appointed and will talk to both parties and anyone else relevant. The investigator's role is to:</p>
+
+            <ul>
+                <li>establish what happened, in a fair and objective manner</li>
```
Comment (dbooth-boston) [inline conflict_resolution.html:169]: At some point a determination must be made about whether the CoC was violated.  Based on this draft, it sounds like this responsibility lies with the investigator.  Therefore I suggest adding another bullet here:

`<li>determine whether the incident involved a violation of the CoC</li>`

File: conflict_resolution.html [hunk 26]
```diff
@@ -0,0 +1,200 @@
...
+        <section id="disciplinary-actions">
+            <h2>Disciplinary Actions</h2>
+
+            <p>There are various actions that the investigator can recommend, depending on the frequency and severity of the behaviours in conflict with the CoC.</p>
+
+            <p>The intent of the conflict resolution procedures is to ensure that individuals can continue to participate in W3C activities.</p>
+
+            <p>Where it is beneficial to the continued functioning of W3C, a high level summary of results and actions from an investigation may be communicated to the membership, for example if an individual is suspended from a working group or from being a chair in any working group. Otherwise, results and actions should be kept confidential.</p>
+
+            <p>Chairs, ombudspeople, mediators, and investigators can recommend the following actions be taken if resolution is not possible:</p>
```
Comment (dbooth-boston) [inline conflict_resolution.html:187]: Other text in this document says that mediators must remain neutral and must not state an opinion, so this sentence seems to be in conflict with that guidance.  That conflict needs to be resolved.

Comment (dbooth-boston) [reply conflict_resolution.html:187]: Also, I don't think we should tie W3M's or the investigator's hands about what actions may be recommended or imposed.  I think it should say something like "recommended actions may include . . . ".

File: conflict_resolution.html (hunk 24 above)
Comment (dbooth-boston) [reply conflict_resolution.html:190]: Another option: s/The individual who had the complaint raised against them/The offender/.

The text will be simpler if we just refer to the person who violated the CoC as "the offender".

File: conflict_resolution.html [hunk 27]
```diff
@@ -0,0 +1,200 @@
...
+
+            <ol>
+                <li>Corrective action. The individual who had the complaint raised against them will be advised to correct their behaviour going forward, for example not engaging in the same behaviours that resulted in the complaint.</li>
+                <li>Agree to a mediation process. The individual who had the complaint raised against them will be advised to participate in a mediation process, where both parties can agree to certain actions and changes going forward. </li>
```
Comment (dbooth-boston) [inline conflict_resolution.html:191]: s/individual who had the complaint raised against them/offender/

File: conflict_resolution.html [hunk 28]
```diff
@@ -0,0 +1,200 @@
...
+                <li>An official warning. In cases where the actions by an individual are frequent or severe, an individual and their employer (if relevant) will be given an official warning that their conduct is contrary to the CoC and that they should immediately desist in those behaviours. If they continue, this may result in the removal or suspension of their W3C participation. There is no time limit for the expiry of the warning.</li>
```
Comment (dbooth-boston) [inline conflict_resolution.html:192]: s/actions by an individual/actions by an offender/

s/an individual and their employer/the offender and their employer/

s/they continue/the behaviors/

File: conflict_resolution.html [hunk 29]
```diff
@@ -0,0 +1,200 @@
...
+                <li>Disciplinary meeting. In cases where the actions by an individual are frequent or severe, and where greater intervention is needed, an individual and a representative from their place of employment (if relevant) will be asked to attend a meeting with W3M. They will be told where their conduct is contrary to the CoC and that they should immediately desist in those behaviours. If they continue, they may result in the removal or suspension of their W3C participation.</li>
```
Comment (dbooth-boston) [inline conflict_resolution.html:193]: s/the actions by an individual/an offender's CoC violation(s)/

s/an individual and a representative/the offender and a representative/

s/They will be told/The offender will be told/

s/they continue/the behaviors/

s/their W3C participation/the offender's W3C participation/

File: conflict_resolution.html [hunk 30]
```diff
@@ -0,0 +1,200 @@
...
+                <li>Temporary removal from a working group, or as a chair of a working group or task force facilitator. In cases where the actions by an individual has been frequent and disruptive, or they have failed to adhere to actions agreed through mediation, they will be suspended from participating in any working groups, chair positions, or as a taskforce facilitator for a period of time agreed on by decision makers.</li>
```
Comment (dbooth-boston) [inline conflict_resolution.html:194]: s/the actions by an individual has/the offender's violations have/

s/they have failed/the offender has failed/

s/they will be suspended/the offender will be suspended/

s/agreed on by decision makers/determined by W3M/

File: conflict_resolution.html [hunk 31]
```diff
@@ -0,0 +1,200 @@
...
+                <li>Final written warning or suspension. Where individuals have received multiple warnings, across the same or multiple parts of the CoC. The individual and their employer will receive a written warning that any further complaints will result in the removal of their W3C participation. They may also be suspended from W3C for a given length of time.</li>
```
Comment (dbooth-boston) [inline conflict_resolution.html:195]: I suggest deleting the "Final written warning" item.  How many warnings does someone need?   A prior bullet already says that a warning may be given, and it does not restrict the number of warnings that may be given.  I think it should be up to W3M's discretion to decide how many warnings are reasonable for a given offender, based on the offender's history and the nature of the violations.  A "final warning" is unnecessary.  Let's not tie W3M's hands.

File: conflict_resolution.html [hunk 32]
```diff
@@ -0,0 +1,200 @@
...
+                <li>Termination of participation. Where there have been exceptional circumstances or frequent upheld complaints against an individual, they may have their participation in W3C terminated. The individual and their employer will receive a letter from the W3C outlining the reasons and why termination was warranted.</li>
```
Comment (dbooth-boston) [inline conflict_resolution.html:196]: s/exceptional circumstances or frequent upheld complaints against an individual/egregious or repeated CoC violations/

s/they may have/the offender may have/

s/The individual/The offender/

**Review by dbooth-boston — COMMENTED** [review]
Overall, I think this is making very good progress.   I think it generally has the right components, but would benefit from editorial work.  I made numerous editorial suggestions accordingly.

File: conflict_resolution.html (hunk 18 above)
Comment (rachelyager-nyc) [reply conflict_resolution.html:88]: @dbooth-boston, I completely agree that it's important for the community to know that they can report any incidents. Additionally, it's crucial that the management team at W3C regularly reviews the analytics and makes them transparent to both the community and the board. By doing so, we can ensure that the PWE is given the priority it deserves instead of being treated as an afterthought.

Comment (rachelyager-nyc) [reply conflict_resolution.html:88]: @wareid @dbooth-boston @TzviyaSiegman I suggest creating sections on Incident Reporting Guidance, Report Management oversights, and Analytics Transparency for the W3C members.
- ### Guidance to Reporting an Incident.
- ### Routine W3M Management Review of Incident Reporting
- ### Transparency of Analytics to W3C Community

Individuals' privacy and confidentiality can be protected by sharing analytics instead of actual reports.

File: conflict_resolution.html (hunk 17 above)
Comment (rachelyager-nyc) [reply conflict_resolution.html:86]: +1 I agree. It's essential to follow the best practices that allow individuals to resolve issues themselves before seeking external help. As per the project management best practice (PMBOK), individuals should be given a chance to resolve conflicts on their own. In our case, the Ombudsperson should remain impartial and use the PWE resources and Conflict Management Procedure to help the individuals initiate the conflict resolution process.

**Comment by rachelyager-nyc** [comment]
@wareid After reviewing the modifications, I am comfortable merging this version and reviewing the updated one in the next PWETF meeting.

**Comment by dbooth-boston** [comment]
> @rachelyager-nyc @dbooth-boston I have made some changes in accordance with your feedback. I believe more work is needed, but if you're happy with the current state, I'd like to merge this so we can review the much more readable version and solicit more feedback.

I suggest incorporating as many of the existing comments as you can, and then merge so that we can see how it looks and reassess.

File: conflict_resolution.html (hunk 18 above)
Comment (TzviyaSiegman) [reply conflict_resolution.html:88]: I don't think it's a good idea to create guidance for WHEN to report an incident, because there are things that might SEEM unimportant but do actually merit reporting. In terms of analytics and incident reporting metrics, this will be covered by ombuds training. We need to work with the Team to assess who it is exposed to.

File: conflict_resolution.html (hunk 7 above)
Comment (wareid) [reply conflict_resolution.html:65]: Created issue #359 to cover this question.

File: conflict_resolution.html (hunk 15 above)
Comment (wareid) [reply conflict_resolution.html:140]: Created issue #360 to cover conversation on this.

**Review by prettychicagoboy1 — APPROVED** [review]
conflict_resolution.html
